packages/dev-server/index.js

```diff
@@ -47,7 +47,11 @@ function postConfig (context, util) {
         hot: true,
         hotOnly: true,
         historyApiFallback: true,
-        inline: true
+        inline: true,
+        // Disable verbose loggin in browser’s console
+        clientLogLevel: 'none',
```

Comment:
I'd rather prefer `warn` or at least `error` here. I don't always look at the console running webpack. The on-page console is open pretty much all-the-time instead.